cminject.jl

```diff
@@ -75,15 +80,31 @@ function run_argparse(args)
         "-f"
             arg_type = String
             help = "Path of a flow-field HDF5 file"
-            required = true
```

Comment:
why??

Reply:
Because in order to allow Stark simulations, the flow field shouldn't be required anymore: This allows Stark simulations without a flow field (which, for now, is the only possible simulation).